src/componentsBasic/home/homeSections/homeSubscribe/HomeSubscribe.js

```diff
@@ -1,8 +1,14 @@
+import React, { useState } from "react";
+
 import "./HomeSubscribe.scss";
 import inkstand from "../../../../assets/images/inkstand.png";
-import Button from '../../../../componentsUniversal/button/Button';
+import Button from "../../../../componentsUniversal/button/Button";
```

Comment:
you can setup jsconfig file with default path. It will help you to avoid path to top level, i mean ../../../../../../. If you set up it correct it will looks like import Button from 'componentsUniversal/button/Button'